package.json

```diff
@@ -10,6 +10,7 @@
     "libEs6"
   ],
   "bin": {
+    "alloy": "scripts/tui-builder.js",
     "build-alloy": "scripts/build-alloy.js"
```

Comment:
If we just provide one executable, then it is much more simple to run because npx can determine which to run. What if both used the same script, but you could pass an --interactive flag?